scripts/build-package.js

```diff
@@ -2,40 +2,35 @@
 
 /* eslint-disable global-require */
 const { resolve } = require('path');
+const execa = require('execa');
 const terminalSize = require('window-size');
-const { checkDependenciesAndRun, spawn } = require('./utils/cli-utils');
 
-const getStorybookPackages = () => {
-  const listCommand = spawn(`lerna list`, {
-    stdio: 'pipe',
-  });
+const getStorybookPackages = async () => {
+  const { stdout } = await execa.command(`yarn workspaces list --json`);
```

Comment:
not using `lerna` here anymore, perhaps over time we can remove it?

Seems like yarn workspaces & `nx` should be able to do everything `lerna` is doing?